From 1a489831b39d0c1583bd4d5e8583f65c1ba880e0 Mon Sep 17 00:00:00 2001
From: Emmanuele Bassi <ebassi@gnome.org>
Date: Wed, 10 May 2017 14:31:09 +0100
Subject: build: Fix the check on sassc's presence

If sassc is not available we do not want to include the rules to
regenerate the CSS out of the SCSS files.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 05fdf4e..4e68e2d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1821,7 +1821,7 @@ AC_SUBST(GDK_HIDDEN_VISIBILITY_CFLAGS)
 
 AC_PATH_PROG([SASSC], [sassc])
 AC_ARG_VAR(SASSC)
-AM_CONDITIONAL([REBUILD_SCSS], [test "xSASSC" != x])
+AM_CONDITIONAL([REBUILD_SCSS], [test "x$SASSC" != x])
 
 ##################################################
 # Output commands
-- 
cgit v0.12

